src/util/dateHelpers.js

```diff
@@ -2,10 +2,36 @@
 
 import { LocalDate, ChronoUnit } from "js-joda";
 
+const months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+var days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+
 export function getToday(): LocalDate {
   return LocalDate.now();
 }
 
+export function getDayOfWeek(): LocalDate {
+  return days[new Date().getDay()];
```

Comment:
Can you use JS Jodatime instead of the built-in JavaScript Date() as we are currently using JS Jodatime in this project due to  https://js-joda.github.io/js-joda/cheat-sheet.html

```
// obtain the current date in the system default timezone, e.g. 2016-02-23
LocalDate.now();

// obtain the current date in the utc timezone, e.g. 2016-02-23
LocalDate.now(ZoneOffset.UTC);```

> Why yet another javascript date and time library
> Popular javascript date libraries like moment or date-utils are wrappers around the native javascript Date object, providing syntactic sugar. The native Date object always consist of a date, time and a timezone part. In opposite to that, js-joda is a standalone date and time implementation.
> 
> The API has a domain-driven design with classes for the different use cases, like LocalDate, ZonedDateTime or Period. For examples LocalDate allows to handle dates like birthdays or holidays in a clean and error-safe way, especially if these dates are persisted to an external server.
> 
> js-joda is immutable. Immutability aligns well with pure functions and with the architecture of frameworks like React and Flux.

Reply:
Sure. I was wondering if there was going to be a date lib used. I will get this updated later today!

Reply:
Thank you!